tests/e2e/runners/firefox/firefox70.bs.runner.js

```diff
@@ -33,5 +33,5 @@ async function loadFirefoxDriver () {
 const driver_gutenberg_fx = await loadFirefoxDriver();
 
 // Run test in SW mode only
-console.log('Running tests in Service Worker mode only for this browser version');
-await gutenbergRo.runTests(driver_gutenberg_fx);
+console.log('Running Gutenberg tests in ServiceWorker mode only for this browser version');
```

Comment:
Test claimed to be for SW mode only, but in fact it was running both JQuery and SW. In the interests of saving testing time, I've made it conform to the description. It was also only for Gutenberg.